index.js

```diff
@@ -1,7 +1,9 @@
 const messageService = require('./server/services/message-service')
+const healthService = require('./server/services/health-service')
 
 async function startService () {
   await messageService.registerQueues()
+  setInterval(healthService, 10000)

```

Comment:
It would be better to have the check interval read from an environment variable so both the health service and the healthz script are reading the same value

Reply:
I was debating this with myself this morning.  Agree it would be better.  The healthz script having the same value is a typo, it should have been 30000.  That would have given a safety buffer to prevent the pod being shut down prematurely.

Reply:
Moved to variable.  Variable documented in readme.